src/nuts.py

```diff
@@ -22,16 +22,14 @@ def nuts():
 
 @nuts.command()
 @click.argument("path_to_shapes")
-@click.argument("path_to_attributes")
 @click.argument("path_to_output")
-def merge(path_to_shapes, path_to_attributes, path_to_output):
-    """Merge NUTS shapes with attributes."""
+def to_multipolygon(path_to_shapes, path_to_output):
+    """Map NUTS shapes to multipolygon."""
     shapes = gpd.read_file(path_to_shapes)
-    shapes.geometry = shapes.geometry.map(_to_multi_polygon)
-    attributes = gpd.read_file(path_to_attributes)
-    attributes = pd.DataFrame(attributes) # to be able to remove geo information
-    del attributes["geometry"]
-    shapes.merge(attributes, on="NUTS_ID", how="left").to_file(path_to_output, driver=OUTPUT_DRIVER)
+    shapes.geometry = shapes.geometry.buffer(0).map(_to_multi_polygon)
```

Comment:
The buffer(0) can have unwanted side effects. Are you sure it's necessary? It wasn't necessary for NUTS 2013, was it? Is it maybe only necessary for some NUTS within [2006, 2010, 2013, 2016]?

Reply:
I assume it's because some shapes are invalid. Maybe a conditional buffering would be better here, so as to not touch all the shapes?

Example: https://github.com/calliope-project/solar-and-wind-potentials/blob/eda64bcc3e143f8203710d1254b2bee973cfea0d/src/shared_coast.py#L72